src/sentry/runner/commands/run.py

```diff
@@ -777,17 +777,35 @@ def handler(signum, frame):
 @run.command("ingest-replay-recordings")
 @log_options()
 @configuration
-@kafka_options("ingest-replay-recordings")
+@kafka_options(
+    consumer_group="ingest-replay-recordings",
+    include_batching_options=True,
+    default_max_batch_size=20,
+)
 @click.option(
     "--topic", default="ingest-replay-recordings", help="Topic to get replay recording data from"
 )
+@click.option("--processes", default=1, type=int)
+@click.option("--input-block-size", type=int, default=DEFAULT_BLOCK_SIZE)
+@click.option("--output-block-size", type=int, default=DEFAULT_BLOCK_SIZE)
 def replays_recordings_consumer(**options):
     from sentry.consumers import print_deprecation_warning
 
     print_deprecation_warning("ingest-replay-recordings", options["group_id"])
```

Comment:
the replays-recordings consumer now uses unified CLI for options. this entire command does no longer run in production. PTAL at `src/sentry/consumers/__init__.py`

Reply:
@untitaker Is this enough? https://github.com/getsentry/sentry/pull/48276/files#diff-e5ee943f84dc1d6870ab01db646066449461e8f35ebdfab60048a4b3b6f73c59R108